IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -169,6 +177,13 @@ - (void)logWithLevel:(MSIDLogLevel)level
     });
 }
 
+- (id)traceFile:(__unused NSString *)filename
```

Comment:
should this be implemented? what's the intended functionality?

Reply:
The functionality in MSAL C++ is to log on function entrance and exit (it logs on destroy). It's a way to deal with stack traces crossing language boundaries, as well as to track the execution flow.
I'm assuming you aren't interested in this functionality -- it does create rather a lot of log messages. However, it should be part of the interface so that we can interact with it.